scripts/qa_crowbarsetup.sh

```diff
@@ -5366,6 +5381,7 @@ function onadmin_qa_test
 # $cct_skip_func_tests -> optional, functional tests will be skipped if value is 1, default is 0
 function onadmin_run_cct
 {
+    pre_hook $FUNCNAME
```

Comment:
it is not only needed in the cct step. We need to exclude the admin crowbar IP and the public endpoint vip, right at the beginning. E.g. for tempest it is creating an image and wants to talk to the public vip.